src/routes/people/TeamPanel.svelte

```diff
@@ -81,7 +83,7 @@
                             : 'bg-csi-white text-csi-black dark:bg-blue-white'}
                     <li class="m-0 p-0">
                         <button
-                            class="min-h-10 shrink-0 rounded-md px-6 shadow-md {neutral} transition-colors"
+                            class="min-h-10 shrink-0 rounded-md px-6 shadow-md transition-colors duration-300 hover:bg-csi-blue dark:hover:bg-csi-black dark:hover:text-csi-white {neutral}"
```

Comment:
Colors nitpick, but personally I don't think `dark:hover:bg-csi-black` fits here. I think a more muted lighter color like `dark:hover:bg-white` would look nicer.

```suggestion
                            class="min-h-10 shrink-0 rounded-md px-6 shadow-md transition-colors duration-300 hover:bg-csi-blue dark:hover:bg-csi-white {neutral}"
```

Reply:
Gets! I wasn't sure about the colors too; I'll commit the changes along with changes to the durations.